documentation/src/main/asciidoc/introduction/Interacting.adoc

```diff
@@ -1061,7 +1064,7 @@ if (!firstPage.isLastPage()) {
 ----
 
 The "key" in key-based pagination refers to a unique key of the result set which determines a total order on the query results.
-In this example, `Book.isbn` is the key.
+In this example, `Book_.isbn` is the key.
```

Comment:
@NathanQingyangXu please revert this one. The key is the `isbn` field of `Book`, not its representation in the metamodel.

Reply:
reverted back